discovery/spec.md

```diff
@@ -304,6 +308,44 @@ The following sections define the attributes that appear in a Service entity.
 - Examples:
   - `http://cloud.example.com/docs/blobstorage`
 
+##### deprecated
+
+- Type: Object containing 2 properties:
+  - sunsettime<br>
+    An OPTIONAL property indicating the time when the Service MAY be removed.
```

Comment:
Given that we don't use the word "sunset" anywhere in the description, maybe "removaltime"? That also avoids the easily-typo'd double t.

Reply:
Possibly earliestremovaltime to indicate that it's a lower bound...